contracts/feirari/RewardsDistributorAdmin.sol

```diff
@@ -4,15 +4,20 @@ import "../refs/CoreRef.sol";
 import "./IRewardsDistributorAdmin.sol";
 import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
 
+/// @notice this contract has its own internal RBAC. The reason for doing this
```

Comment:
Not familiar with this acronym. ACL is the one I am more familiar with.

Reply:
Role Based Access Control